.eslintrc

```diff
@@ -235,6 +234,8 @@
     "prefer-reflect": 1,
     "prefer-spread": 1,
     "require-yield": 2,
-    "react/jsx-no-bind": 2
+    "react/jsx-no-bind": [2, {
+      "ignoreRefs": true
+    }]
   }
```

Comment:
Please do not make ESLint changes. We have a company wide ruleset on all our projects.